src/room.py

```diff
@@ -1,2 +1,16 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+from item import Item
+
+class Room:
+    def __init__(self, name, description):
+        self.name = name
+        self.description = description
+        self.items = [
+            Item("candle", "it gets dark at night"),
+            Item("sock", "smells bad but keeps the feet warm"),
+            Item("some food item", "always hungry")
+        ]
```

Comment:
Try to have the items list as something you can instantiate in the constructor (in __init__) .. otherwise every room is going to start with a candle, a sock, and some food item.  If you set that in the contstructor you can instantiate each room with its own unique item list.